.github/workflows/build.yml

```diff
@@ -1,13 +1,43 @@
-name: ioserver
+name: client-build
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Hey @t1m0thyj,
Sorry fro going back to this... 😢 
Do we want to enable the build on the `main` branch (on `push`)?